02.calendar/calendar.rb

```diff
@@ -0,0 +1,25 @@
+require 'optparse'
+require 'date'
+
+options = ARGV.getopts('y:', 'm:')
+year = options['y']
+year ||= Date.today.year
+month = options['m']
+month ||= Date.today.month
+
+def calendar(year, month)
+  year = year.to_i
+  month = month.to_i
+  puts "      #{month}月 #{year}"
+  puts "日 月 火 水 木 金 土"
+  (1..Date.new(year, month, -1).day).each do |day|
+    date = Date.new(year, month, day)
+    date.cwday.times { print "\s" * 3 } if day == 1
+    print "\s" if day / 10 == 0
+    print "#{ date == Date.today ? "\e[30;47m#{day}\e[0m" : day }"
+    print "\s"
+    print "\n" if date.saturday?
+  end
+end
+
+calendar(year, month)
```

Comment:
```suggestion
calendar(year.to_i, month.to_i)
```

として、calendarメソッドは数値のyearとmonthしか受け取らない、とした方がメソッドの設計として一貫性があっていいかも、と思いました。

Reply:
おおお、なるほど！ メソッド実行時に `to_i `すると、受け取り方を限定するという意味合いになるんですね！ 
👀 からウロコです〜！
そして、そうすると def 内のこの2行も消せて、良き……！

https://github.com/taea/ruby-practices/pull/4/commits/ef74d62fea61f8731b1b3c2766a76f55a7e4ae72